DEVELOPER.md

```diff
@@ -133,7 +112,15 @@ cd cloud-platform-deployment-manager
 make && DEBUG=yes make docker-build
 ```
 
+With Golang version 1.17.9, the source directory is not mandatory in ${HOME}/go/src.
+You can create any directory except inside GOPATH and run after the step "git clone" above.
```

Comment:
```suggestion
You can create any directory except inside GOPATH and run after the step "git clone" above.
```
I would say something like this (please correct me if it's changing the meaning):

You can create a directory anywhere except under GOPATH. and run the "git clone" step described above.